### ISSUE TITLE
DOC: Google Chrome flags documentation pages as dangeorus

### ISSUE THREAD

**Issue by shaoran** [pandas-dev/pandas#39905]
#### Location of the documentation

If you go to https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.drop.html with a current Google Chrome (Version 88.0.4324.150 (Official Build) (64-bit), Linux), you get this red page

![dangerous](https://user-images.githubusercontent.com/113313/108495010-88cadb00-72a8-11eb-9eae-90b0dfd2cf3e.png)

that says that the site is dangerous. Upon further inspection, you come across [this entry of the transparency report](https://transparencyreport.google.com/safe-browsing/search?url=https:%2F%2Fpandas.pydata.org%2Fpandas-docs%2Fstable%2Freference%2Fapi%2Fpandas.DataFrame.drop.html&hl=en-US) which says:

> The site https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.drop.html contains harmful content, including pages that:
>
> * Install unwanted or malicious software on visitors’ computers
> * Contain unwanted or malicious software

If you tell Chrome that you understand the risks, then it displays the page but a dangerous icon is still being displayed at the URL box.

A similar thing happens with Firefox (`78.7.1esr`).

**Comment by MarcoGorelli** [comment]
thanks, duplicate of #39862

**Comment by AveCoders** [comment]
I've recently done a tutorial on Pandas and it was blocked by YouTube with the appeal been rejected on the grounds that I'm violating community guidelines by producing harmful content. Any suggestions?

**Comment by MarcoGorelli** [comment]
@AveCoders I don't know, perhaps if you ask on the Gitter channel (link in README) someone will know
